Kickstarter-iOS/Views/Cells/CommentCell.swift

```diff
@@ -141,6 +141,21 @@ final class CommentCell: UITableViewCell, ValueCell {
         guard let self = self else { return }
         self.delegate?.commentCellDidTapViewReplies(self, comment: comment)
       }
+
+    self.viewModel.outputs.shouldIndentContent
+      .observeForUI()
+      .observeValues { shouldIndent in
+        _ = self.rootStackView
+          |> \.axis .~ .vertical
+          |> \.layoutMargins .~ .init(
+            top: Styles.grid(1),
+            left: Styles.grid(shouldIndent ? 5 : 1),
```

Comment:
What does 5 and 1 represent here? It's a magic number unless we add documentation or store it as a constant somewhere.